sparse/_compressed/compressed.py

```diff
@@ -248,7 +206,13 @@ def __str__(self):
 
     def change_compressed_axes(self, new_compressed_axes):
         """
-        changes the compressed axes of an array.
+        Changes the compressed axes of this array. This operation is similar to converting 
+        a scipy.sparse.csc_matrix to a scipy.sparse.csr_matrix.
+
+        Returns
+        -------
+        GCXS
+            The output array.
```

Comment:
Can we document that this doesn't modify in-place but returns a transposed version.